VideoLocker/res/values/strings.xml

```diff
@@ -505,6 +505,9 @@
     <string name="enrollment_failure">We could not enroll you in this course at this time.</string>
     <!-- Alert dialog button allowing user to repeat their attempted action -->
     <string name="try_again">Try again</string>
+    <!-- Registration error -->
+    <string name="registration_error_title">Registration Error</string>
+    <string name="registration_error_message">Please verify the information that was entered.</string>
```

Comment:
@mdinino Just a small tweak. How about "Please verify the information that you entered."